### PR TITLE
♻️ Rename (get|update)Priority to (get|update)LayoutPriority

### PR DESCRIPTION
**Pull request by danielrozenberg**
Partial fix for #13555

### CODE REVIEW

File: src/service/resources-impl.js [hunk 1]
```diff
@@ -1967,11 +1967,11 @@ export class Resources {
       if (resource.getState() == ResourceState.NOT_BUILT) {
         resource.whenBuilt().then(() => {
           this.measureAndScheduleIfAllowed_(resource, layout,
-              parentResource.getPriority());
+              parentResource.getLayoutPriority());
```
Comment (choumx) [inline src/service/resources-impl.js]: Looks like priority is also used for preload in addition to layout. I'm okay with this change but another option is to just update the jsdoc to more clearly spell this out in custom-element.js and base-element.js.

**Comment by choumx** [comment]
Update the PR description please.

File: src/service/resources-impl.js (hunk 1 above)
Comment (jridgewell) [reply src/service/resources-impl.js]: Hm? It's used for preloading, but specifically for layout in preloading.

**Comment by danielrozenberg** [comment]
Update: after an offline discussion, restored the method renames and kept the comment

**Pull request merged**